src/NetscriptFunctions/Sleeve.ts

```diff
@@ -238,8 +238,10 @@ export function NetscriptSleeve(): InternalAPI<NetscriptSleeve> {
       const action = helpers.string(ctx, "action", _action);
       checkSleeveAPIAccess(ctx);
       checkSleeveNumber(ctx, sleeveNumber);
+      let ctract = undefined;
```

Comment:
Please change this to `contractName` and the other one to `contractEnum`. Other names would work too, as long as they are distinguishing between them and don't have ctractions. :D